### PR TITLE
Bug 1905338: updated add-flow-scenarios automation scripts

### PR DESCRIPTION
**Pull request by makambalaji**
Jira Id : 0DC-5186
Bugzilla Id: 1905338
Cypress scripts related to Database, Yaml, Operator-Backed and Page-Details and tag names
Please Note: This scripts are not integrated with CI
If need to verify, execute the below command "npm run test-cypress-devconsole" on fronted

Test Results:
No. of smoke test cases in this pr : 2 [Execution Time: - 51 sec]
No. of regression test cases in this pr: [Execution Time: - sec]
No. of test cases in this pr: [Execution Time: - sec]

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/getting-started/getting_started _to_dev_perspective.feature [hunk 1]
```diff
@@ -2,10 +2,10 @@ Feature: Login to developer perspective
...
-And clicks login
-Then user redirects to developer perspective
\ No newline at end of file
+    @regression
+    Scenario: Developer perspective display on login to open shift application using developer credentials
+        Given user is at login page
+        When user enters the username as "username"
+        And password as "password"
+        And clicks login
+        Then user redirects to developer perspective
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/getting-started/getting_started _to_dev_perspective.feature:11]: Can you please add new line?

**Review by gajanan-more — CHANGES_REQUESTED** [review]
Can you please update the files with suggested changes?

File: frontend/packages/dev-console/integration-tests/features/getting-started/getting_started _to_dev_perspective.feature (hunk 1 above)
Comment (makambalaji) [reply frontend/packages/dev-console/integration-tests/features/getting-started/getting_started _to_dev_perspective.feature]: updated

**Comment by andrewballantyne** [comment]
@makambalaji Would appear you have some type errors. Please look to fix those asap so we can see this merge before the end of the sprint.

**Comment by makambalaji** [comment]
closing this pr, due to merge and labels issue. Creating a fresh pr: https://github.com/openshift/console/pull/7989

**Pull request closed without merging**